.github/workflows/ci.yml

```diff
@@ -131,7 +131,121 @@ jobs:
 
     # run PEcAn checks
     - name: check
-      run: make -j1 check
+      run: make -j1 check_base
```

Comment:
I bet we could further reduce code duplication here by setting the check command as a matrix variable, but that can be a separate PR -- what you have here is already a big improvement!

Reply:
Thank you :smile: